Sources/LiCore/Networking/SSOHandler.swift

```diff
@@ -23,9 +23,21 @@ class SSOHandler: RequestRetrier {
     private let lock = NSLock()
     private var isRefreshing = false
     private var requestsToRetry: [RequestRetryCompletion] = []
+    // MARK: - RequestAdapter
+    func adapt(_ urlRequest: URLRequest) throws -> URLRequest {
+        if LiSDKManager.shared().authManager.isUserLoggedIn() {
+            var urlRequest = urlRequest
+            let accessToken = LiSDKManager.shared().authState.accessToken ?? ""
```

Comment:
should it return a request even when you don't have an access token?

Reply:
The request is still valid, the server will return a 401 though in that case.

Reply:
We can modify the error and catch it. I'll try that.